native/src/routes/Pois.tsx

```diff
@@ -194,11 +194,11 @@ const Pois = ({ pois, language, path, cityModel, navigateTo, navigateToFeedback,
           selectedFeature={selectedFeature}
           locationPermissionGranted={location !== null}
           onRequestLocationPermission={requestAndDetermineLocation}
-          fabPosition={sheetSnapPointIndex < snapPoints.length - 1 ? snapPoints[sheetSnapPointIndex]! : 0}
+          fabPosition={snapPoints[sheetSnapPointIndex] ?? 0}
```

Comment:
```suggestion
           fabPosition={sheetSnapPointIndex < snapPoints.length - 1 ? snapPoints[sheetSnapPointIndex]! : 0}
```

pls revert according to this issue

![image](https://user-images.githubusercontent.com/37902063/148925960-d05e7634-044b-426b-ab2f-2362f9b93898.png)

Reply:
Thx for spotting, I should really not do these kind of changes anymore :disappointed: